### DISCUSSION TITLE
submit

### DISCUSSION THREAD

**Discussion by Paul-Cui** [PaddlePaddle/PaddleSpatial#84] [Q&A] [answered]
Dear organizer,
     Which demo should we refer to if we want to test the submitted results?  Cause there are code files with the same name  in  two folders (wpf_baseline_gru and kddcup22-sdwpf-evaluation) ，but they seem to be different versions of code.

Thanks!

**Comment by ramber1836** [comment, marked as answer]
Please refer to the latest version of codes that are located in the folder
kddcup22-sdwpf-evaluation.
Thanks.

Regards,
Xinjiang Lu

**Reply by Paul-Cui** [reply]
Thanks! In addition, what does test-1.zip mean? Is it a submitted demo?

**Reply by ramber1836** [reply]
Yes.
There is a manual on the submissioin, please refer to it (PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/README.md).

**Reply by Paul-Cui** [reply]
Thanks.
